### DISCUSSION TITLE
Button to fix error of missing Utility upon Tools Restore

### DISCUSSION THREAD

**Discussion by givihuda** [dbeaver/dbeaver#13759] [Ideas]
I've been having trouble figuring out what to do here:

![image](https://user-images.githubusercontent.com/74785256/131723130-e1ab3937-2d9d-47a8-b839-30eef248cfdb.png)

I could do a Google search and dig through settings or whatever, but a button that takes me to the place to fix this... would be nice.

I think the problem started after I ran `brew install whatever`, which updated mysql@5.7 to a sub-minor version :facepalm: before I knew that would happen, which broke the driver or utility in DBeaver, which I'm now having trouble figuring out how to fix.

I think it's in Preferences > Connections > Drivers, but that doesn't even list the current one applied, nor am I seeing exactly what to click at this point. Also I know any button to open Finder will not show /usr folder, but that may be beyond the scope of this post's idea.

**Comment by givihuda** [comment]
I also found that if I click Local Client on the first screen of the restore, I can select the new Cellar/mysql@5.7 version. It just is not clear or obvious what to do until I do a Google search or click around to see what happens (which can be dangerous).

Even just instructions on what to do or where to check for a fix would be _much_ better than the current error message.

The word "utility" shows nothing in Preferences search! So why does the error say it can't find the "Utility" when it's actually talking about the Local Client?! Maybe they're the same thing, but why name them differently? Maybe I could have inferred in the error message that it meant Local Client because it mentioned "client" and "/local" folder, but... I shouldn't have to be Sherlock Holmes to fix an error.

**Comment by LonwoLonwo** [comment]
Hello @givihuda 

Have you tried to look here: https://github.com/dbeaver/dbeaver/issues?q=is%3Aissue+utility+%27mysql%27+not+found+is%3Aclosed ?

**Reply by givihuda** [reply]
That's just a search for 'mysql' 😆  so I don't know which ticket you want me to look at or you're not understanding my last comment. The point was that the interface doesn't make it clear or easy what to do to fix the problem. I had to do a Google search and end up here to know what to do. That's incredibly inefficient, and the names not being consistent across views add to the problem. I only posted the feedback so others could leave their reactions or maybe someone would get the idea to fix it. There's no need to show me more links with more searches, I already figured out the solution, but the underlying problem was UI/UX being unnecessarily difficult.
